### PR TITLE
Improve Invitation flow

### PR DESCRIPTION
**Pull request by MrFlashAccount**
#### Tl;dr 
Closes: enso-org/cloud-v2#1186
This PR significantly improves the invitation UX and add ability to view/resend/copy/delete invitations

<details><summary>Demo Presentation</summary>
<p>

https://github.com/enso-org/enso/assets/61194245/62124243-50ce-47e1-bcac-789463b5e755

</p>
</details> 

---

#### This Change:
What the change does in the larger context? Specific details to highlight for review:
1. Redesign the Invitation dialog
2. Add the ability to edit emails after typing ` `
3. Adds ability to use different separators: `<space>`, `<semicolon>`, `<colon>` or `<newline>`
4. Shows Invitation on the `members` page in settings.
5. Adds ability to remove, resend, copy or delete invitations
6. Improve the UI of dialogs, buttons

#### Test Plan:
Go over how you plan to test it. Your test plan should be more thorough the riskier the change is. For major changes, I like to describe how I E2E tested it and will monitor the rollout.

---

### Checklist

Please ensure that the following checklist has been satisfied before submitting the PR:

- [ ] The documentation has been updated, if necessary.
- [ ] Screenshots/screencasts have been attached, if there are any visual changes. For interactive or animated visual changes, a screencast is preferred.
- [ ] All code follows the
      [Scala](https://github.com/enso-org/enso/blob/develop/docs/style-guide/scala.md),
      [Java](https://github.com/enso-org/enso/blob/develop/docs/style-guide/java.md),
      [TypeScript](https://github.com/enso-org/enso/blob/develop/docs/style-guide/typescript.md),
      and
      [Rust](https://github.com/enso-org/enso/blob/develop/docs/style-guide/rust.md)
      style guides. In case you are using a language not listed above, follow the [Rust](https://github.com/enso-org/enso/blob/develop/docs/style-guide/rust.md) style guide.
- [ ] Unit tests have been written where possible.

### CODE REVIEW

File: app/ide-desktop/lib/dashboard/src/modals/InviteUsersModal/InviteUsersSuccess.tsx [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+ */
+const MANY_EMAILS = 4
+
+/**
+ * Success screen for the invite users modal.
+ */
+export function InviteUsersSuccess(props: InviteUsersSuccessProps) {
+  const { onClose, emails, invitationLink } = props
+  const { getText, locale } = textProvider.useText()
+  const membersSearchParams = 'cloud-ide_page="settings"&cloud-ide_SettingsTab="members"'
```
Comment (somebody1234) [inline app/ide-desktop/lib/dashboard/src/modals/InviteUsersModal/InviteUsersSuccess.tsx]: not sure about this one, it looks pretty fragile. also it should probably be a top level constant if it stays

File: app/ide-desktop/lib/dashboard/src/modals/InviteUsersModal/InviteUsersSuccess.tsx [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+              onPressStart={onClose}
+              href={membersPageUrl}
+            >
+              {getText('goToMembersPage')}
+            </ariaComponents.Button>
+          )}
+
+          <ariaComponents.Button
+            variant="primary"
+            size={isUserOnMembersPage ? 'medium' : 'medium'}
```
Comment (somebody1234) [inline app/ide-desktop/lib/dashboard/src/modals/InviteUsersModal/InviteUsersSuccess.tsx]: huge thonk

File: app/ide-desktop/lib/dashboard/src/modals/InviteUsersModal/InviteUsersSuccess.tsx [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+  readonly emails: string[]
+  readonly invitationLink: string
+}
+
+/**
+ * The number of emails to display in the success message.
+ * If the number of emails is greater than this number, the message will be
+ * x emails have been invited instead of listing all the emails.
+ */
+const MANY_EMAILS = 4
```
Comment (somebody1234) [inline app/ide-desktop/lib/dashboard/src/modals/InviteUsersModal/InviteUsersSuccess.tsx]: constants should be above everything else

File: app/ide-desktop/lib/dashboard/src/modals/InviteUsersModal/InviteUsersForm.tsx [hunk 4]
```diff
@@ -0,0 +1,198 @@
...
+  const getEmailsFromInput = eventCallbackHooks.useEventCallback((value: string) => {
+    return parserUserEmails.parseUserEmails(value)
+  })
+
+  const highlightEmails = eventCallbackHooks.useEventCallback((value: string): void => {
+    if (inputRef.current?.firstChild != null) {
+      const trimValue = value.trim()
+      const { entries } = getEmailsFromInput(value)
+
+      CSS.highlights.delete('field-wrong-email')
```
Comment (somebody1234) [inline app/ide-desktop/lib/dashboard/src/modals/InviteUsersModal/InviteUsersForm.tsx]: please use feature detection, FF doesn't support `CSS.highlights` yet

File: app/ide-desktop/lib/dashboard/src/layouts/Settings.tsx [hunk 5]
```diff
@@ -67,7 +68,7 @@ export default function Settings() {
     })()
   }, [sessionType, backend])
 
-  let content: JSX.Element
+  let content: React.JSX.Element
```
Comment (somebody1234) [inline app/ide-desktop/lib/dashboard/src/layouts/Settings.tsx:75]: unnecessary change, `JSX.Element` is required by TS to exist when using JSX, so it's more framework-agnostic (not that that matters)

File: app/ide-desktop/lib/dashboard/src/components/Loader.tsx [hunk 6]
```diff
@@ -2,28 +2,76 @@
...
-  readonly size?: number
+  readonly size?: Sizes | number
+  readonly state?: spinnerModule.SpinnerState
 }
 
-const DEFAULT_SIZE = 32
+/**
+ * The possible sizes for a {@link Loader}.
+ */
+export type Sizes = 'large' | 'medium' | 'small'
```
Comment (somebody1234) [inline app/ide-desktop/lib/dashboard/src/components/Loader.tsx]: `Sizes` -> `Size`

File: app/ide-desktop/lib/dashboard/src/components/Loader.tsx [hunk 7]
```diff
@@ -2,28 +2,76 @@
...
-import * as tw from 'tailwind-merge'
+import * as twv from 'tailwind-variants'
 
 import Spinner, * as spinnerModule from '#/components/Spinner'
 
 /**
  * Props for a {@link Loader}.
  */
-export interface LoaderProps {
+export interface LoaderProps extends twv.VariantProps<typeof STYLES> {
```
Comment (somebody1234) [inline app/ide-desktop/lib/dashboard/src/components/Loader.tsx:12]: nit: `twv.VariantProps<typeof STYLES>` could be its own type

File: app/ide-desktop/lib/dashboard/src/components/Loader.tsx [hunk 8]
```diff
@@ -2,28 +2,76 @@
  * @file
  * A full-screen loading spinner.
  */
-import * as tw from 'tailwind-merge'
+import * as twv from 'tailwind-variants'
```
Comment (somebody1234) [inline app/ide-desktop/lib/dashboard/src/components/Loader.tsx:5]: nit: i'd probably `twVariants` over `twv`. or `tv` i guess, to match the import we are using

File: app/ide-desktop/lib/dashboard/src/components/AriaComponents/index.ts [hunk 9]
```diff
@@ -2,7 +2,9 @@
  * @file index.ts
  * Index file for Aria Components
  */
-export * from './Button/Button'
+export * from './Button'
 export * from './Tooltip/Tooltip'
```
Comment (somebody1234) [inline app/ide-desktop/lib/dashboard/src/components/AriaComponents/index.ts:6]: is there a reason we deduplicated `Button`'s path but not `Tooltip`'s?

File: app/ide-desktop/lib/dashboard/src/components/AriaComponents/Inputs/ResizableInput/ResizableContentEditableInput.tsx [hunk 10]
```diff
@@ -0,0 +1,133 @@
...
+
+    const onPaste = eventCallbackHooks.useEventCallback(
+      (event: React.ClipboardEvent<HTMLDivElement>) => {
+        // Prevent pasting styled text.
+        event.preventDefault()
+        // sanitize the pasted text
+        // replace all < with &lt; to prevent XSS
+        const text = event.clipboardData
+          .getData('text/plain')
+          .replace(/</g, '&lt;')
```
Comment (somebody1234) [inline app/ide-desktop/lib/dashboard/src/components/AriaComponents/Inputs/ResizableInput/ResizableContentEditableInput.tsx:68]: prefer extracting `escapeHTML` into `string.ts`:
```ts
const REPLACEMENTS: Record<string, string> = {
  '&': '&amp;', '<': '&lt;', '>': '&gt;', '"': '&quot;', '\'': '&#039;'
}

function escapeHTML(unsafe: string) {
  return unsafe.replace(/[&<>"']/g, m => REPLACEMENTS[m] ?? m);
}
```

File: app/ide-desktop/lib/dashboard/src/components/AriaComponents/CopyBlock/CopyBlock.tsx [hunk 11]
```diff
@@ -0,0 +1,65 @@
...
+ */
+
+import * as React from 'react'
+
+import * as twv from 'tailwind-variants'
+
+import * as ariaComponents from '#/components/AriaComponents'
+
+/**
+ *
```
Comment (somebody1234) [inline app/ide-desktop/lib/dashboard/src/components/AriaComponents/CopyBlock/CopyBlock.tsx:14]: blank docs

File: app/ide-desktop/lib/dashboard/src/components/AriaComponents/Button/Button.tsx [hunk 12]
```diff
@@ -37,106 +36,102 @@ interface PropsWithoutHref {
...
+      custom: '',
+      hero: 'px-8 py-4 text-lg',
+      large: 'px-6 py-3 text-base',
+      medium: 'px-4 py-2 text-sm',
+      small: 'px-3 py-1 text-xs',
+      xsmall: 'px-2 py-1 text-xs',
+      xxsmall: 'px-1.5 py-0.5 text-xs',
+    },
+    iconOnly: { true: '' },
+    rounding: {
```
Comment (somebody1234) [inline app/ide-desktop/lib/dashboard/src/components/AriaComponents/Button/Button.tsx]: i'd probably prefer `rounded='lg'` (for example) to match the tailwind classnames to avoid having to learn a naming convention that is *slightly* different, but similar enough that it'd end up confusing the user

File: app/ide-desktop/lib/dashboard/src/components/AriaComponents/Button/Button.tsx [hunk 13]
```diff
@@ -37,106 +36,102 @@ interface PropsWithoutHref {
...
-   */
-  readonly size?: Size
-  readonly rounding?: Roundings
+  readonly icon?: string | null
   /**
-   * Button takes the full width of its container
+   * When `true`, icon will be shown only when hovered.
    */
-  readonly fullWidth?: boolean
+  readonly showIconOnHover?: boolean
```
Comment (somebody1234) [inline app/ide-desktop/lib/dashboard/src/components/AriaComponents/Button/Button.tsx:49]: do we actually want this? this sounds like it will cause the size of the element to change

File: app/ide-desktop/lib/dashboard/src/services/remoteBackendPaths.ts [hunk 14]
```diff
@@ -27,6 +27,7 @@ export const DELETE_ORGANIZATION_PATH = 'organizations/me'
 export const UPLOAD_ORGANIZATION_PICTURE_PATH = 'organizations/me/picture'
 /** Relative HTTP path to the "invite user" endpoint of the Cloud backend API. */
 export const INVITE_USER_PATH = 'users/invite'
+export const INVITATION_PATH = 'invitations'
```
Comment (somebody1234) [inline app/ide-desktop/lib/dashboard/src/services/remoteBackendPaths.ts:33]: missing docs

File: app/ide-desktop/lib/dashboard/src/tailwind.css [hunk 15]
```diff
@@ -237,6 +237,8 @@
 
   --menu-entry-padding: 0.25rem;
 
+  --delete-color: rgba(243 24 10 / 87%);
```
Comment (somebody1234) [inline app/ide-desktop/lib/dashboard/src/tailwind.css:240]: `rgba` -> `rgb`

File: app/ide-desktop/lib/dashboard/src/tailwind.css [hunk 16]
```diff
@@ -237,6 +237,8 @@
...
+  --cta-color: #fa6c08;
```
Comment (somebody1234) [inline app/ide-desktop/lib/dashboard/src/tailwind.css]: prefer `rgb()` for consistency

File: app/ide-desktop/lib/dashboard/src/services/RemoteBackend.ts [hunk 17]
```diff
@@ -254,15 +254,58 @@ export default class RemoteBackend extends Backend {
 
   /** Invite a new user to the organization by email. */
   override async inviteUser(body: backend.InviteUserRequestBody): Promise<void> {
-    const path = remoteBackendPaths.INVITE_USER_PATH
```
Comment (somebody1234) [inline app/ide-desktop/lib/dashboard/src/services/RemoteBackend.ts:257]: why remove `path`?

File: app/ide-desktop/lib/dashboard/src/modals/InviteUsersModal/parseUserEmails.ts [hunk 18]
```diff
@@ -0,0 +1,36 @@
...
+ * removing any empty strings and should support emails copied from gmail, outlook, apple mail.
+ * Supports emails in the following formats:
+ *
+ * ***NOTE*** This function does not validate the email addresses, it only parses them.
+ * @returns an object with the parsed entries and any errors that occurred during parsing.
+ */
+export function parseUserEmails(emailsString: string): ParseUserEmailsResult {
+  const entries = emailsString
+    .split(/[,;\s\n]+/)
+    .map(entry => entry.trim())
```
Comment (somebody1234) [inline app/ide-desktop/lib/dashboard/src/modals/InviteUsersModal/parseUserEmails.ts:32]: optional: you can use `.flatMap()` instead of `.map().filter().map()`

File: app/ide-desktop/lib/dashboard/src/modals/InviteUsersModal/InviteUsersSuccess.tsx [hunk 19]
```diff
@@ -0,0 +1,97 @@
...
+    search: membersSearchParams,
+  })
+  const { search } = reactRouter.useLocation()
+
+  const emailListFormatter = React.useMemo(
+    () => new Intl.ListFormat(locale, { type: 'conjunction', style: 'long' }),
+    [locale]
+  )
+
+  const isUserOnMembersPage = decodeURIComponent(search) === `?${membersSearchParams}`
```
Comment (somebody1234) [inline app/ide-desktop/lib/dashboard/src/modals/InviteUsersModal/InviteUsersSuccess.tsx]: is there a reason this isn't passed as a `boolean` prop?

File: app/ide-desktop/lib/dashboard/src/components/Loader.tsx [hunk 20]
```diff
@@ -2,28 +2,76 @@
...
+ * The possible sizes for a {@link Loader}.
+ */
+export type Sizes = 'large' | 'medium' | 'small'
+
+const STYLES = twv.tv({
+  base: 'animate-appear-delayed flex h-full w-full items-center justify-center duration-200',
+  variants: {
+    minHeight: {
+      full: 'h-full',
+      h6: 'h-6',
```
Comment (somebody1234) [inline app/ide-desktop/lib/dashboard/src/components/Loader.tsx:28]: prefer `6` or `h-6` over `h6`

**Review by somebody1234 — APPROVED** [review]
CR passed (other than minor issues raised in comments).
QA not yet done

**Comment by somebody1234** [comment]
uhh

**Comment by somebody1234** [comment]
QA :x: ("`CSS.highlights` is undefined" crashing the entire app the moment i press any key in the invite modal)

File: app/ide-desktop/lib/dashboard/src/modals/InviteUsersModal/InviteUsersSuccess.tsx (hunk 1 above)
Comment (MrFlashAccount) [reply app/ide-desktop/lib/dashboard/src/modals/InviteUsersModal/InviteUsersSuccess.tsx]: Yeah, it's heavily fragile, and atm I see 2 solutions: either refactor settings to routes or use `@tanstack/router` since it provides typesafety for URLs

File: app/ide-desktop/lib/dashboard/src/modals/InviteUsersModal/InviteUsersSuccess.tsx (hunk 2 above)
Comment (MrFlashAccount) [reply app/ide-desktop/lib/dashboard/src/modals/InviteUsersModal/InviteUsersSuccess.tsx]: 😂 🙈

**Comment by somebody1234** [comment]
also remember to change `Spaces` to `spaces`

File: app/ide-desktop/lib/dashboard/src/modals/InviteUsersModal/InviteUsersForm.tsx (hunk 4 above)
Comment (MrFlashAccount) [reply app/ide-desktop/lib/dashboard/src/modals/InviteUsersModal/InviteUsersForm.tsx]: Yea, that's why we wrapped it in try/catch block. I think this should be sufficient, WDYT?

File: app/ide-desktop/lib/dashboard/src/modals/InviteUsersModal/InviteUsersSuccess.tsx (hunk 1 above)
Comment (somebody1234) [reply app/ide-desktop/lib/dashboard/src/modals/InviteUsersModal/InviteUsersSuccess.tsx]: i left a suggestion somewhere below suggesting to use an explicit `boolean` prop instead. might not be *ideal*, but at the very least it should be less fragile right?

File: app/ide-desktop/lib/dashboard/src/layouts/Settings.tsx (hunk 5 above)
Comment (MrFlashAccount) [reply app/ide-desktop/lib/dashboard/src/layouts/Settings.tsx:75]: `JSX.Element` is deprecated by `@types/react`

File: app/ide-desktop/lib/dashboard/src/modals/InviteUsersModal/InviteUsersForm.tsx (hunk 4 above)
Comment (somebody1234) [reply app/ide-desktop/lib/dashboard/src/modals/InviteUsersModal/InviteUsersForm.tsx]: oh didn't see that. but this one is not in a try catch so it is crashing Firefox

File: app/ide-desktop/lib/dashboard/src/layouts/Settings.tsx (hunk 5 above)
Comment (somebody1234) [reply app/ide-desktop/lib/dashboard/src/layouts/Settings.tsx:75]: o_O

File: app/ide-desktop/lib/dashboard/src/components/AriaComponents/index.ts (hunk 9 above)
Comment (MrFlashAccount) [reply app/ide-desktop/lib/dashboard/src/components/AriaComponents/index.ts:6]: mostly because the tooltip doesn't have a barrel import and Button now has(because it exposes now more that just Button, also - ButtonGroup, CopyButton, CloseButton)

File: app/ide-desktop/lib/dashboard/src/components/AriaComponents/Button/Button.tsx (hunk 13 above)
Comment (MrFlashAccount) [reply app/ide-desktop/lib/dashboard/src/components/AriaComponents/Button/Button.tsx:49]: Afair we use opacity to hide the icon. Ideally, I guess we might want to provide an external `twv variants`  scheme that satisfies the button's one(e.g. that extends the button's), but I still have no clue how to do that properly. By doing so, we can customize any slot and no longer need to use these properties (it's less safe though)

File: app/ide-desktop/lib/dashboard/src/services/RemoteBackend.ts (hunk 17 above)
Comment (MrFlashAccount) [reply app/ide-desktop/lib/dashboard/src/services/RemoteBackend.ts:257]: AFAIR I inlined the varible

File: app/ide-desktop/lib/dashboard/src/modals/InviteUsersModal/parseUserEmails.ts (hunk 18 above)
Comment (MrFlashAccount) [reply app/ide-desktop/lib/dashboard/src/modals/InviteUsersModal/parseUserEmails.ts:32]: Not sure they're identical... `flatMap` unfolds nested arrays, but here our array is already flat

File: app/ide-desktop/lib/dashboard/src/components/Loader.tsx (hunk 20 above)
Comment (MrFlashAccount) [reply app/ide-desktop/lib/dashboard/src/components/Loader.tsx:28]: IMO h6 is easier to type and it doesn't violate our rules regarding keys in objects. WDYT?

**Comment by somebody1234** [comment]
QA :x: 
- Note: QA performed on Electron as the modal is currently still crashing on Firefox.
- :heavy_check_mark: Emails in the "invite users" modal are now editable
- :heavy_check_mark: Pending invites are visible in the "members" settings tab
- :warning: No warning when typing two of the same email in the input box. Maybe it shouldn't block sending invites, but IMO it should still be highlighted - e.g. in case the user copy-pasted an email because they plan to fill in the rest later (e.g. `.group@example.com`)
- :warning: "Spaces" -> "spaces, commas or semicolons" - or "spaces, `,` or `;`"
- :x: The font size of the success screen is way too big:
  - :warning: The entire list is surrounded by quotes. Either each individual email should be surrounded by quotes, or there should be no quotes at all.
- :information_source: Cannot mass delete invites. Maybe we should have a button to clear all pending invites?
  - Marked as :information_source: because multi-select would require a major refactor and hence is probably out of scope.

image
- :x: The modal cannot be closed by clicking on the dimmed background
- :x: Username should be a separate column in the "members" table in the settings
- :x: You should not be able to remove yourself from the organization
  - :warning: Remove button should probably also always be red
- :x: No dimmed background for "invite" modal in "members" settings tab
  - :heavy_check_mark: Clicking outside the modal hides the modal
- :warning: Ideally, deleted invitations would be removed from the list optimistically
  - :warning: Deleting multiple invitations at once feels laggy

Minor UI nits:
- :warning: The font size for the title is tiny for some reason
- :warning: we should maybe make the close button red on hover? see the dashboard-122 branch (or related PR) for the specific color that is currently being used
- The border radius of the buttons are a lot lower than those of the buttons everywhere else
  - The height of the buttons is also inconsistent

File: app/ide-desktop/lib/dashboard/src/services/RemoteBackend.ts (hunk 17 above)
Comment (somebody1234) [reply app/ide-desktop/lib/dashboard/src/services/RemoteBackend.ts:257]: yeah i mean that works, but separate `path` just makes it consistent with all the other endpoints in this file

File: app/ide-desktop/lib/dashboard/src/modals/InviteUsersModal/parseUserEmails.ts (hunk 18 above)
Comment (somebody1234) [reply app/ide-desktop/lib/dashboard/src/modals/InviteUsersModal/parseUserEmails.ts:32]: the `flatMap` would replace the `filter` - although it's true that it may be less readable if done like that

File: app/ide-desktop/lib/dashboard/src/components/Loader.tsx (hunk 20 above)
Comment (somebody1234) [reply app/ide-desktop/lib/dashboard/src/components/Loader.tsx:28]: makes sense, but considering it's used as a value in all other code i think it's fine to do an eslint-disable here. but of course this is mostly just personal preference, i just think `h6` looks a bit weird

File: app/ide-desktop/lib/dashboard/src/modals/InviteUsersModal/InviteUsersForm.tsx (hunk 4 above)
Comment (MrFlashAccount) [reply app/ide-desktop/lib/dashboard/src/modals/InviteUsersModal/InviteUsersForm.tsx]: yeah, will take a look

**Comment by MrFlashAccount** [comment]
Addressing your QA remarks:

> ❌ Username should be a separate column in the "members" table in the settings

Why though? 

> ❌ You should not be able to remove yourself from the organization

Currently, this button does nothing, I think we can tune up the behavior in this issue: https://github.com/enso-org/cloud-v2/issues/890

> ❌ No dimmed background for "invite" modal in "members" settings tab

Popovers do not dim background because basically, they are tooltips with rich content.

> we should maybe make the close button red on hover

Not sure. I think generally we should use red color when we want to highlight irreversible actions like "Remove" or cancel. Closing the dialog isn't one of them.

**Comment by somebody1234** [comment]
> Username should be a separate column
> why?

for consistency - none of our other tables have multiline cells

> Currently, this button does nothing

we have a dedicated button elsewhere for deleting the current user, and it comes with plenty of warning

> Popovers do not dim background

in that case we might want an actual modal instead - the "invite users" dialog is something that:
- should take the user's focus, and
- should either be addressed immediately (or closed immediately)

> Closing the dialog isn't an irreversible action

oh, i meant the traffic light in the top left, so that it matches the macos traffic lights

File: app/ide-desktop/lib/dashboard/src/components/Loader.tsx (hunk 7 above)
Comment (MrFlashAccount) [reply app/ide-desktop/lib/dashboard/src/components/Loader.tsx:12]: Not sure on usecases. Suppose we can leave it for now.

File: app/ide-desktop/lib/dashboard/src/modals/InviteUsersModal/InviteUsersSuccess.tsx (hunk 1 above)
Comment (MrFlashAccount) [reply app/ide-desktop/lib/dashboard/src/modals/InviteUsersModal/InviteUsersSuccess.tsx]: Not sure, the url string will always be a string :)

File: app/ide-desktop/lib/dashboard/src/tailwind.css (hunk 15 above)
Comment (MrFlashAccount) [reply app/ide-desktop/lib/dashboard/src/tailwind.css:240]: Not sure we can easily replace this color with RGB only alternative, since it heavily depends on a background. Actually, I copied this color(and all below) from tailwind.config.js so I guess we can leave it as is?

File: app/ide-desktop/lib/dashboard/src/components/AriaComponents/Inputs/ResizableInput/ResizableContentEditableInput.tsx (hunk 10 above)
Comment (MrFlashAccount) [reply app/ide-desktop/lib/dashboard/src/components/AriaComponents/Inputs/ResizableInput/ResizableContentEditableInput.tsx:68]: Not sure. the code you provided scales better for sure, but I doubt we need to replace anything expect `<>` symbols...

**Comment by MrFlashAccount** [comment]
> for consistency - none of our other tables have multiline cells

We'll have then an empty cell for invitations, which is strange. 

> in that case we might want an actual modal instead - the "invite users" dialog is something that:

Okay

> oh, i meant the traffic light in the top left, so that it matches the macos traffic lights

Good idea 👍

**Comment by MrFlashAccount** [comment]
> No warning when typing two of the same email in the input box. Maybe it shouldn't block sending invites, but IMO it should still be highlighted - e.g. in case the user copy-pasted an email because they plan to fill in the rest later (e.g. .group@example.com)

We just ignore duplicates, not sure we need to warn users...

**Comment by MrFlashAccount** [comment]
> we have a dedicated button elsewhere for deleting the current user, and it comes with plenty of warning

Thanks for noticing, will take a look once when we'll work on this feature. Ideally, I think, we should merge the behavior.
But for invitations - I think it's safe if we delete without confirmation (We can create a new invitation anytime)

**Comment by MrFlashAccount** [comment]
I think I've addressed most of the issues and PR is ready for re-QA

**Review by PabloBuchu — APPROVED** [review]

**Comment by PabloBuchu** [comment]
ready to merge after fixing conflitcs

**Comment by PabloBuchu** [comment]
1. Go to members page is not working (nothing happens)
2. Secondly if `GET /invites` is not ending with success it hangs Enso. I am on solo so backend responds with 403 but if I reload I dont see Enso only periodically re issued get invites request

https://github.com/enso-org/enso/assets/2855109/8977a404-a928-4fd3-9903-bf4fb6c5e961

**Review by PabloBuchu — CHANGES_REQUESTED** [review]
see comment

**Review by PabloBuchu — APPROVED** [review]

File: app/ide-desktop/lib/dashboard/src/tailwind.css (hunk 15 above)
Comment (somebody1234) [reply app/ide-desktop/lib/dashboard/src/tailwind.css:240]: `rgba` is an obsolete syntax,  `rgb(243 24 10 / 87%);` is completely valid

File: app/ide-desktop/lib/dashboard/src/tailwind.css [hunk 21]
```diff
@@ -237,6 +237,8 @@
 
   --menu-entry-padding: 0.25rem;
 
+  --delete-color: rgba(243 24 10 / 87%);
+  --cta-color: rgb(250, 108, 8);
```
Comment (somebody1234) [inline app/ide-desktop/lib/dashboard/src/tailwind.css:241]: this causes issues with overriding opacity
